virginia.clubrunning.org/join.html

```diff
@@ -208,6 +208,10 @@ <h2>Social</h2>
                 <li>
                     <p>We host parties every 2-3 weeks.</p>
                 </li>
+                <li>
+                    <p>Frolicsome takes the place of regular practice on Fridays. Come play other sports (soccer,
```

Comment:
Nit: I would write "Frolicsome Fridays take the place of regular Friday practice."
Also, capitalize "Club Runners"